frame/staking/src/benchmarking.rs

```diff
@@ -471,6 +471,10 @@ benchmarks! {
 
 		let era = <Staking<T>>::current_era().unwrap_or(0);
 		let caller: T::AccountId = account("caller", n, SEED);
+
+		// Whitelist caller account from further DB operations.
+		let caller_key = frame_system::Account::<T>::hashed_key_for(&caller);
+		frame_benchmarking::benchmarking::add_whitelist(caller_key);
```

Comment:
note that this key will be added by each instanciation of benchmark.

I.e. if someone run a benchmark with repeat = 100, 3 components with 10 steps, we instance 3*10*100 times thus whitelist will contains 3_000 same keys, maybe this should be fixed by modifying add_whitelist which should check that the key doesn't already exist, or by modifying set_whitelist in client/db/bench in order deduplicate the keys